### ISSUE TITLE
Model template should not put unnecessary quotes around string value 

### ISSUE THREAD

**Issue by tfesenko** [RepreZen/KaiZen-OpenAPI-Editor#83]
Our model template puts quotes around string values, but code completion does not (See #41 Code assist should not put unnecessary quotes around string value) . It makes it inconsistent:
<img width="154" alt="screen shot 2016-01-21 at 9 55 32 am" src="https://cloud.githubusercontent.com/assets/644582/12483831/23dea0cc-c025-11e5-99b4-113c729bdcc8.png">

**Comment by tedepstein** [comment]
Good catch, @tfesenko !  

Here's a corrected template, without the quotes:

```
---
# This is a sample Swagger spec, describing a simple API as a starting point.  
swagger: "2.0"
info:
  description: Tax Blaster
  version: 1.0.0
  title: TaxBlaster
host: taxblaster.com
basePath: /api
schemes:
- http

# Tags organize operations into groups for presentation in the Swagger UI.
# Each tag has an optional description, which the Swagger UI will display in 
# the tag group header.
tags:
- name: TaxFilingObject
  description: An individual Tax Filing record, accessed by its ID

paths:

  # Each Path Item Object describes a resource, containing a set of operations
  # at a specified path.  The Path Item object can define parameters and 
  # responses common all of its contained operations.
  /taxFilings/{id}:

    # Operations are identified by an HTTP method.  
    get:
      tags:
      - TaxFilingObject
      description: 
      operationId: getTaxFiling
      consumes:
      - application/json
      produces:
      - application/json
      parameters:
      - name: id
        in: path
        description: filingID of the requested TaxFiling
        required: true
        type: string
      responses:
        200:
          description: Non-empty description
          schema:
            $ref: TaxFilingObject
          examples:
            # Example message, keyed by media type
            application/json :
              {
                filingID : 1234,
                jurisdiction : Federal,
                year : 2015,
                period : CALENDAR_YEAR,
                currency : EUR,
                grossIncome : 74832,
                taxLiability : 15640
              }
        404:
          description: 

# The definitions section contains a set of named Schema Objects.  Each schema
# object describes a reusable data type, which can be reference by name.
definitions:
  TaxFilingObject:
    type: object
    description: An individual Tax Filing record, accessed by its ID
    properties:
      filingID:
        type: string
      jurisdiction:
        type: string
      year:
        type: string
      period:
        type: integer
      currency:
        type: string
      grossIncome:
        type: number
      taxLiability:
        type: number
      taxpayer:
        type: object
```

**Comment by tfesenko** [comment]
Fixed in 2abad62

**Comment by tedepstein** [comment]
@tfesenko , your comment says this was fixed, and the commit you referenced shows this.  So I'm closing it.
